tmp/build/library-sync.sh

```diff
@@ -13,13 +13,18 @@ pushd library-master
 rm -rf arch community* .git* hack import_content.py LICENSE Makefile official* OWNERS requirements.txt .travis.yml
 pushd operator
 rm -rf okd-x86_64
-rm -rf ocp-ppc64le
 pushd ocp-x86_64
 pushd official
 mv * ..
 popd # official
 rmdir official
 popd # ocp-x86_64
+pushd ocp-ppc64le
+pushd official
+mv * ..
+popd # official
+rmdir official
+popd #ocp-ppc64le
```

Comment:
What about `ocp-s390x`?

Reply:
interestingly I still see a s390x subfolder but will address / re-run thx

Reply:
updated thanks

Reply:
right, without this you get e.g. `ocp-s390x/official/perl` instead of just `ocp-$ARCH/perl` like the other arches.